vmware_aria_operations_integration_sdk/docker_wrapper.py

```diff
@@ -43,9 +43,16 @@
 logger.addHandler(consoleHandler)
 
 
-def login(docker_registry: str, **kwargs: Any) -> str:
-    print(f"Login into {docker_registry}")
-    command = ["docker", "login", f"{docker_registry}"]
+def login(**kwargs: Any) -> str:
+    command = ["docker", "login"]
+    if "docker_registry" not in kwargs:
+        print(f"Login into Docker Hub")
+        docker_registry = "docker.io"
+    else:
+        docker_registry = kwargs["docker_registry"]
+        print(f"Login into {docker_registry}")
+        command.append(docker_registry)
+
```

Comment:
this is called with keyword 'container_registry' now, but we're still checking for 'docker_registry' here.